src/vl-mapactions-custom-map.js

```diff
@@ -115,21 +128,7 @@ export class VlCustomMap extends VlMapWithActions {
   }
 
   initializeView(boundingBox, maxZoom) {
-    if (!this.getView().getZoom()) {
-      const view = new View({
-        extent: this.projection.getExtent(),
-        projection: this.projection,
-        maxZoom: 16,
-        minZoom: 2,
-        center: [140860.69299028325, 190532.7165957574],
-        zoom: 2,
-      });
-      this.zoomViewToExtent(view, boundingBox, maxZoom);
-      this.setView(view);
-      if (this.overviewMapControl) {
-        this.addControl(this.overviewMapControl); // control needs to be added after view initialization
-      }
-    }
+    this.zoomViewToExtent(this.view, boundingBox, maxZoom);
```

Comment:
Kan vervangen worden door `this.getView()` zodat de view const declaratie overbodig wordt.